wall_traps.md

```diff
@@ -199,3 +199,13 @@ you're not careful:
 
 [Back to top](#top)
 </div>
+
+## HoH Hall of Fallacies
+
+<div class="surfacePane" markdown="1">
+
+* Any outer wall can have a trap:
```

Comment:
I would expect that there are specific places that traps may be, like on other floors, and also that this location would be specific to the floor 1-19 map. Do you have reason to suspect otherwise?

Reply:
I don't have enough data to say either way - I took this screenshot after hitting a wall trap once and wanting confirmation that it really was a wall trap rather than just sloppy movement, so that's only two cases, and I haven't taken the time to look for more.

I'm tempted to say "given the special layout, they may have simply specified the entirety of each map segment as a spawn location, rather than taking the time to choose specific spawn areas", on the basis that (1) as a programmer, that would probably be my first implementation choice for that sort of layout, and (2) anecdotally, I've seen traps in separate map segments extremely close to each other, closer than I would expect for manually specified spawn areas - though I didn't think to take a screenshot at the time. But again, that's all guesswork and I don't have the data to say one way or the other.

Not sure how you'd prefer to describe the possibility of these wall traps, but feel free to close this PR and make use of the screenshot yourself if you like.

Reply:
Those are good points too... I will investigate a little and get back to you! 🏃